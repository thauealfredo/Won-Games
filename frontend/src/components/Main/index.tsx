import * as STL from './styles';

const Main = () => (
  <STL.Wrapper>
    <STL.Logo
      src="/img/logo.svg"
      alt="Imagem de um atÃ³mo - escrito ao lado Won Games"
    />
    <STL.Title>Won Games</STL.Title>
    <STL.Description>
      TypeScript, ReactJS, NextJS, Styled Components
    </STL.Description>
    <STL.Logo
      src="/img/hero-illustration.svg"
      alt="Desenvolvedor na frente do comptutador"
    />
  </STL.Wrapper>
);

export default Main;
