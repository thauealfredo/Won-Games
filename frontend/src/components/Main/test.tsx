import { screen, render } from '@testing-library/react';

import Main from '.';

describe('<Main />', () => {
  it('should render the heading', () => {
    const { container } = render(<Main />);

    expect(
      screen.getByRole('heading', { name: /won games/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
