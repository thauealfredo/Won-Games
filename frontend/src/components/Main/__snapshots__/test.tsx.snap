// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<main
  class="styles__Wrapper-sc-18tj86z-0 kvvBEb"
>
  <img
    alt="Imagem de um atÃ³mo - escrito ao lado Won Games"
    class="styles__Logo-sc-18tj86z-1 kqUYFU"
    src="/img/logo.svg"
  />
  <h1
    class="styles__Title-sc-18tj86z-2 gZbkHa"
  >
    Won Games
  </h1>
  <h2
    class="styles__Description-sc-18tj86z-3 fMaWrV"
  >
    TypeScript, ReactJS, NextJS, Styled Components
  </h2>
  <img
    alt="Desenvolvedor na frente do comptutador"
    class="styles__Logo-sc-18tj86z-1 kqUYFU"
    src="/img/hero-illustration.svg"
  />
</main>
`;
